### ISSUE TITLE
hibernate-validator - CVE-2020-10693 (Improper Input Validation)

### ISSUE THREAD

**Issue by y4ppieflu** [OSSIndex/vulns#164]
A flaw was found in Hibernate Validator version 6.1.2.Final. A bug in the message interpolation processor enables invalid EL expressions to be evaluated as if they were valid. This flaw allows attackers to bypass input sanitation (escaping, stripping) controls that developers may have put in place when handling user-controlled data in error messages.

**Advisory details**
```
  URL: https://nvd.nist.gov/vuln/detail/CVE-2020-10693, https://snyk.io/vuln/SNYK-JAVA-ORGHIBERNATE-568162
  format: maven
  namespace: org.hibernate
  name: hibernate-validator
  versions: <6.0.19.Final, >=6.10<6.1.3.Final
```

**Comment by ken-duck** [comment]
Another problem with a project being associated with multiple CPEs. Fixed and public by tomorrow.

**Comment by wjglerum** [comment]
Seems like 6.2.0 is included now, see https://ossindex.sonatype.org/vulnerability/6360908f-637a-4214-a2e9-fd57263d84c9

However RedHat mentions it's fixed in that version, see https://bugzilla.redhat.com/show_bug.cgi?id=1805501

**Comment by ndonewar** [comment]
Closing this issue as it appears to be resolved and 6.2.0 is no longer included. Please feel free to reopen or create a new issue if needed. Thanks!
